web/docs/guides/with-ionic-react.mdx

```diff
@@ -0,0 +1,640 @@
+---
+id: with-ionic-react
+title: "Quickstart: Ionic React"
+description: Learn how to use Supabase in your Ionic React App.
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+## Intro
+
+This example provides the steps to build a simple user management app (from scratch!) using Supabase and Ionic React. It includes:
+
+- Supabase [Database](/docs/guides/database): a Postgres database for storing your user data.
+- Supabase [Auth](/docs/guides/auth): users can sign in with magic links (no passwords, only email).
+- Supabase [Storage](/docs/guides/storage): users can upload a photo.
+- [Row Level Security](/docs/guides/auth#row-level-security): data is protected so that individuals can only access their own data.
+- Instant [APIs](/docs/guides/api): APIs will be automatically generated when you create your database tables.
+
+By the end of this guide you'll have an app which allows users to login and update some basic profile details:
+
+![Supabase User Management example](/img/ionic-demos/ionic-angular-account.png)
+
+Clicking this button the application will:
```

Comment:
Is this copy pasted from somewhere? I'm not entirely sure which button you mean!